content/news/2024-03-11-kickoff/index.md

```diff
@@ -12,6 +12,8 @@ summary: >
     building a **European Network of Research Software Quality** began.
 ---
 
+{{< figure src="/images/everse-session.png" alt="EVERSE session, Developing and Promoting Research Software Quality in EOSC, at EOSC Symposium 2024" href="https://eosc.eu/symposium2024/" >}}
```

Comment:
Hi @aspaor - I don't understand why you put this image into the news item for the Kickoff meeting? I think that should be removed. Especially as it's out of date since 26 April!

Reply:
Yes, I will remove it. Thank you or noticing that.